### ISSUE TITLE
Operands could not be broadcast together with shapes

### ISSUE THREAD

**Issue by tricoffee** [hamidriasat/DDRNets#2]
Operands could not be broadcast together with shapes (512, 1024, 64) (128, 256, 64)
  # Low to High
  x_temp = layers.Activation("relu")(layers_inside[2])
  x_temp = layers.Conv2D(highres_planes, kernel_size=(1,1), use_bias=False)(x_temp)
  x_temp = layers.BatchNormalization()(x_temp)
  x_temp = tf.image.resize(x_temp, (height_output, width_output)) # 1/16 -> 1/8
  x_ = layers.Add()([x_, x_temp]) # next high branch input, 1/8

**Comment by tricoffee** [comment]
Are you sure your code can work ???????????????????????????????????????????

**Comment by hamidriasat** [comment]
Hi tricoffee,

**Comment by hamidriasat** [comment]
Hi tricoffee,

I have confirmed my code is working with input size of [1024, 2048,3] and [512, 1024, 3].
if you are facing problem's kindly share your complete settings so that i can recreate the error on my end. Also make sure you are using TensorFlow 2.0 or later version.

Thanks

**Comment by hamidriasat** [comment]
Not responding.
